frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -26,14 +26,17 @@ import { EmotionTheme, hasLightBackgroundColor } from "@streamlit/lib"
  * @param theme The theme to use.
  * @returns The horizontal spacing for the sidebar.
  */
-export const getSidebarHorizontalSpacing = (theme: EmotionTheme): string => {
+export const getSidebarHorizontalSpacing = (
+  theme: EmotionTheme,
+  scrollbarGutterSize: number
+): string => {
   // This should be max(0px, ...), but there's a Chrome bug that
   // causes content to clip when scrollbar-gutter is set to "stable both-edges".
-  // So we change the min from 0px to --scrollbar-width to account for that.
+  // So we change the min from 0px to scrollbarGutterSize to account for that.
   // Chrome bug: https://issues.chromium.org/issues/40064879
   return `max(
-    calc(var(--scrollbar-width)),
-    calc(${theme.spacing.lg} - var(--scrollbar-width))
+    ${scrollbarGutterSize}px,
+    calc(${theme.spacing.xl} - ${scrollbarGutterSize}px)
```

Comment:
Changed from `spacing.lg` to `spacing.xl` which is closer to the actual sidebar mockups and works leads to more consistent spacing with different scrollbar sizes.